src/zepben/evolve/database/paths/ewb_data_file_paths.py

```diff
@@ -0,0 +1,247 @@
+#  Copyright 2024 Zeppelin Bend Pty Ltd
+#
+#  This Source Code Form is subject to the terms of the Mozilla Public
+#  License, v. 2.0. If a copy of the MPL was not distributed with this
+#  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+from datetime import date, timedelta
+from pathlib import Path
+from typing import Callable, Iterator, Optional, List
+
+from zepben.evolve import require
+from zepben.evolve.database.paths.database_type import DatabaseType
+
+__all__ = ['EwbDataFilePaths']
+
+
+class EwbDataFilePaths:
+    """Provides paths to all the various data files / folders used by EWB."""
+    _base_dir: Path = None
+    create_directories_func: Callable[[Path], Path] = None
+    is_directory: Callable[[Path], bool] = None
+    exists: Callable[[Path], bool] = None
+    list_files: Callable[[Path], Iterator[Path]] = None
```

Comment:
This is not a dataclass, so probably remove all these as they are class variables rather than instance variables.